applications/vessel-history/src/routes/routes.selectors.ts

```diff
@@ -136,6 +136,11 @@ export const selectHasSearch = createSelector(
 
 export const selectUrlAkaVesselQuery = selectQueryParam<string[]>('aka')
 
+export const selectMergedVesselId = createSelector(
```

Comment:
😆  it aims to return a unique string that combines the vessel Id with it's merged vessel ids to use as key to store the vessel's voyages state... not that you mention it maybe it would be better to use `selectVesselProfileId` instead of `selectVesselId`. I'll try it. Good catch!